extensions/kubeconfig/podlogs.go

```diff
@@ -49,6 +57,17 @@ func GetPodLogs(client *rancher.Client, clusterID string, podName string, namesp
 	defer stream.Close()
 
 	reader := bufio.NewScanner(stream)
+
+	if bufferSizeStr != "" {
+		bufferSize, err := parseBufferSize(bufferSizeStr)
```

Comment:
I'm not completely sure the purpose of this in this function, would you be able to let me know why we need this? can we not proceed without this function?